app/assets/javascripts/tournaments/tournament_pools.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
+    $('.tournament-pool-fighters-sortable').sortable({
```

Comment:
This is a four-space tab, and it should be two.
We should really get linters for this :)

I'm also not a fan of just inlining scripts like that, but we haven't established any guidelines...